$NetBSD: patch-ai,v 1.1 2006/10/29 07:51:28 dsainty Exp $

Under Solaris, PWD is more likely to be set to the working directory of the
Pkgsrc "make" than the working directory, because Solaris /bin/sh does not
support PWD.  An empty PWD will also fail, because configure's "test -z $PWD"
will error if PWD is unset (PWD should be quoted).

http://bugzilla.gnome.org/show_bug.cgi?id=366113

--- configure.old	2006-10-17 16:40:42.000000000 -0400
+++ configure	2006-10-29 00:59:57.000000000 -0400
@@ -432,15 +432,13 @@
 test -f .makeconfig && rm .makeconfig
 test -f .tmakeconfig && rm .tmakeconfig
 
-if test -z $PWD; then
-  PWD=`pwd`
-fi
+configPWD=`pwd`
 
 cat > .makeconfig <<EOF
-DOXYGEN   = $PWD
-TMAKEPATH = $PWD/tmake/lib/$f_platform
+DOXYGEN   = $configPWD
+TMAKEPATH = $configPWD/tmake/lib/$f_platform
 ENV       = env TMAKEPATH=\$(TMAKEPATH)
-TMAKE     = $PWD/tmake/bin/tmake
+TMAKE     = $configPWD/tmake/bin/tmake
 MAKE      = $f_make
 PERL      = $f_perl
 RM        = rm -f
